install_config/install/host_preparation.adoc

```diff
@@ -433,8 +433,7 @@ free space, then run *docker-storage-setup* and review the output to ensure the
 +
 ----
 # cat /etc/sysconfig/docker-storage
-DOCKER_STORAGE_OPTIONS=--storage-opt dm.fs=xfs --storage-opt
-dm.thinpooldev=/dev/mapper/docker--vg-docker--pool
+DOCKER_STORAGE_OPTIONS="--storage-driver devicemapper --storage-opt dm.fs=xfs --storage-opt dm.thinpooldev=/dev/mapper/docker--vg-docker--pool --storage-opt dm.use_deferred_removal=true --storage-opt dm.use_deferred_deletion=true "
```

Comment:
Just a little polish:
"/dev/mapper/docker--vg-docker--pool" is not insistent with the following `lvs` output, so here it should be:
--storage-opt dm.thinpooldev=/dev/mapper/rhel-docker--pool